DEPS

```diff
@@ -46,12 +46,19 @@ hooks = [
     'action': ['vpython3', 'script/download_rust_deps.py', '--platform', 'linux'],
   },
   {
-    # Download rust deps if necessary for the other platforms (macOS and Windows)
+    # Download rust deps if necessary for macOS
     'name': 'download_rust_deps',
     'pattern': '.',
-    'condition': 'not checkout_android and not checkout_ios and not checkout_linux',
+    'condition': 'checkout_mac',
     'action': ['vpython3', 'script/download_rust_deps.py'],
   },
+  {
+    # Download rust deps if necessary for Windows
+    'name': 'download_rust_deps',
+    'pattern': '.',
+    'condition': 'checkout_win',
+    'action': ['vpython', 'script/download_rust_deps.py'],
```

Comment:
this says TEMP, but it's still in there. Why would it stop working on vpython3 and why is the error from go code?

Reply:
This is from https://github.com/brave/brave-core/pull/7866/commits/cc32316047e0a174b17b86d2bb9994f413ef1050 so tagging @mkarolin explicitly as the original author (also, sorry for messing up the authorship of this commit while rebasing earlier today!).

Reply:
The troublemaker appears to be https://chromium.googlesource.com/chromium/src/+/efdcb8407e0d9139e142aaf236cffb97b5e6d7f1%5E%21/#F0 that added the `wheel` for `psutil 5.7.2` to `.vpython3`. But there doesn't appear to be a prebuilt wheel for the particular architecture (`amd64_cp38_cp38m`). Even when I try to use `pip` to install that version of `psutil` it attempts to download source distribution and build locally (which fails since I don't have python-dev installed) instead of grabbing a prebuilt package. Should we use a patch to comment out that `wheel` from the file?